src/coord/src/coord.rs

```diff
@@ -3546,6 +3546,90 @@ impl Coordinator {
                 }
                 explanation.to_string()
             }
+            ExplainStage::Timestamp => {
```

Comment:
If we could break any of this out to a method, and perhaps eventually have `explain.rs` contain all of these, that would be a start towards making this file more manageable.

Reply:
I agree, but for another PR. This timestamp stuff has to be in a coordinator impl so would need a new explain.rs in the coord directory, but yes, we should do...something.